pretty-pipes/pp-lib/src/main/cpp/pp/Hdfs.h

```diff
@@ -48,15 +48,6 @@ class FileStatus
 public:
   FileStatus() {}
 
-  FileStatus(const FileStatus& in)
-  {
-    _accessTime = in._accessTime;
-    _modTime = in._modTime;
-    _len = in._len;
-    _path = in._path;
-    _kind = in._kind;
-  }
-
```

Comment:
Unnecessary copy constructor that does what a default member-wise copy would do anyway.